### PR TITLE
CLOUD-820 fixed swarm response problems

### PR DESCRIPTION
**Pull request by doktoric**
@akanto

### CODE REVIEW

File: orchestrator-api/src/main/java/com/sequenceiq/cloudbreak/orchestrator/SimpleContainerBootstrapRunner.java [hunk 1]
```diff
@@ -34,6 +34,10 @@ public Boolean call() throws Exception {
                 success = true;
                 LOGGER.info("Container started successfully.");
             } catch (Exception ex) {
+                if (ex.getMessage().startsWith("Conflict") && ex.getMessage().contains("You have to delete (or rename) that container to be able to assign")) {
```
Comment (akanto) [inline orchestrator-api/src/main/java/com/sequenceiq/cloudbreak/orchestrator/SimpleContainerBootstrapRunner.java]: Could you log it with WARN level, please?

**Comment by akanto** [comment]
LGTM

**Comment by martonsereg** [comment]
Do not merge it please, I think it's not OK like this but I'll review it

**Comment by martonsereg** [comment]
I think if this try-catch block would be in DockerClientUtil.createContainer it would be good, and the id of the conflicting container should be returned.

**Pull request merged**